### PR TITLE
[Doc] Refactored GUI.md

### PR DESCRIPTION
**Pull request by Vissidarte-Herman**
Related issue = #

### CODE REVIEW

**Comment by Vissidarte-Herman** [comment]
Something went amiss with GitHub?

File: docs/lang/articles/gui/gui_system.md [hunk 1]
```diff
@@ -4,135 +4,162 @@ sidebar_position: 1
...
+## Coordinate system
 
+Each window is built on a coordinate system: the origin is located in the lower-left corner, with the `+x` direction stretching to the right and the `+y` direction stretching upward.
 
-## Paint on a window
-Taichi's GUI supports painting simple geometric objects, such as lines, triangles, rectangles, circles, and text.
+## Display a field or ndarray
 
-:::note
+To display a Taichi field or a NumPy ndarray, call `gui.set_imag``e()`. The method accepts both types as input.
```
Comment (Leonz5288) [inline docs/lang/articles/gui/gui_system.md:43]: Typo here?

File: docs/lang/articles/gui/gui_system.md [hunk 2]
```diff
@@ -4,135 +4,162 @@ sidebar_position: 1
...
 ```python
-gui.circles(pos, radius=3, palette=[0x068587, 0xED553B, 0xEEEEF0], palette_indices=material)
+gui = ti.GUI(res, title, fast_gui=True)
 ```
-draws circles with radius of 1.5 and three different colors differed by `material`, an integer array with the same size as
-`pos`. Each integer in `material` indicates which color the associated circle use (i.e. array [0, 1, 2] indicates these three
-circles are colored separately by the first, second, and third color in `palette`.
 
-![circles](../static/assets/colored_circles.png)
+For this mode to work, ensure that the data passed into `gui.set_imag``e()` is in a display-compatible format. In other words, If it is a Taichi field, ensure that it is one of the following:
```
Comment (Leonz5288) [inline docs/lang/articles/gui/gui_system.md:70]: Typo

**Review by Leonz5288 — APPROVED** [review]
LGTM

**Pull request merged**